airbyte-integrations/connectors/source-zendesk-sell/source_zendesk_sell/source.py

```diff
@@ -0,0 +1,301 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from abc import ABC
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+import re
+import requests
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+from airbyte_cdk.sources.streams.http.requests_native_auth import TokenAuthenticator
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.streams import IncrementalMixin
+
+# Basic full refresh stream
+class ZendeskSellStream(HttpStream, ABC):
+    """
+    This class represents a stream output by the connector.
+    This is an abstract base class meant to contain all the common functionality at the API level e.g: the API base URL, pagination strategy,
+    parsing responses etc..
+    """
+
+    url_base = "https://api.getbase.com/v2/"
+    primary_key = None
+
+    # def __init__(self, config: Mapping[str, Any], **kwargs):
+    #     super().__init__()
+    #     self.start_date = config["start_date"]
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        try:
+            regex_page='[=?/]page[_=/-]?(\d{1,3})'
+            meta_links = response.json().get('meta', {}).get('links')
+            if 'next_page' in meta_links.keys():
+                return {'page' : int(re.findall(regex_page, meta_links['next_page'])[0])}
+        except Exception as e:
+            print(f"{e.__class__} occurred, while trying to get next page information from the following dict {meta_links}") 
```

Comment:
change print to self.logger.error
```suggestion
            self.logger.error(f"{e.__class__} occurred, while trying to get next page information from the following dict {meta_links}") 
```

Reply:
done